Datadog.Trace.Managed.sln.DotSettings

```diff
@@ -54,6 +62,7 @@
 	<s:String x:Key="/Default/CodeStyle/Naming/CSharpNaming/AutoDetectedNamingRules/=StaticReadonly/@EntryIndexedValue">&lt;Policy Inspect="True" Prefix="" Suffix="" Style="AaBb" /&gt;</s:String>
 	<s:String x:Key="/Default/CodeStyle/Naming/CSharpNaming/AutoDetectedNamingRules/=TypeParameters/@EntryIndexedValue">&lt;Policy Inspect="True" Prefix="T" Suffix="" Style="AaBb" /&gt;</s:String>
 	<s:String x:Key="/Default/CodeStyle/Naming/CSharpNaming/AutoDetectedNamingRules/=TypesAndNamespaces/@EntryIndexedValue">&lt;Policy Inspect="True" Prefix="" Suffix="" Style="AaBb" /&gt;</s:String>
+	<s:String x:Key="/Default/CodeStyle/Naming/CSharpNaming/PredefinedNamingRules/=PrivateStaticReadonly/@EntryIndexedValue">&lt;Policy Inspect="True" Prefix="_" Suffix="" Style="aaBb" /&gt;</s:String>
```

Comment:
The naming convention for both `static readonly` and `const` fields is Pascal case, even if they are private.

Reply:
Alright, I changed it back...I imagine then we should bite the bullet and run a formatter over existing code to apply this stuff, since I just sample a dozen locations where this was not the case...not sure I can find a place where a static readonly actually follows this not counting the ones I just updated...maybe I'm just seeing static that are not readonly, but I'm not sure I see why there'd be a difference...that said, I really don't care, just need to know what it should be so we can be consistent about it

Reply:
Yeah, you're not wrong. @bmermet used lower camel case for these, so you'll see a lot of them in the repo. I use upper camel case in new ones going forward (see [Tracer.cs](https://github.com/DataDog/dd-trace-dotnet/blob/master/src/Datadog.Trace/Tracer.cs)), and will usually change the ones I see in a file if I am making other changes there. Otherwise I leave them alone to avoid creating lots of unrelated diffs in PRs. I don't think it's worth going back and changing all of them if we will probably catch them all over time.

StyleCop isn't throwing errors on these, is it?